projects/project_tests.py

```diff
@@ -119,3 +119,71 @@ def test__projects__admin__ProjectResource__1(browser, project_dict):
 def test__projects__models__1(db, project):
     """Project gets represented by its title."""
     assert str(project) == project.title
+
+
+def test__projects__admin__ProjectAdmin__1(browser, project_dict):
+    """Export the projects date to ics."""
+    project_dict['begind_date'] = datetime(
+        year=2022,
+        month=9,
+        day=26,
+        tzinfo=TZ,
+    )
+    project_dict['title'] = 'new_project'
+    proj1 = create_project(project_dict)
+
+    project_dict['begin_date'] = datetime(
+        year=2021,
+        month=9,
+        day=26,
+        tzinfo=TZ,
+    )
+    project_dict['title'] = 'old_project'
+    proj2 = create_project(project_dict)
+
+    browser.login_admin()
+    browser.open(A_PROJ_URL)
+    browser.follow('Export dates')
+
+    assert browser.headers['Content-Type'] == 'text/calendar'
+    assert proj1.title in str(browser.contents)
+    assert proj2.title in str(browser.contents)
+
+
+def test__projects__admin__ProjectAdmin__2(browser, project_dict):
+    """Export the projects date of this year to ics."""
```

Comment:
As you also export the project title, description and end, it would be good to test for that too, maybe in a separate test or via a explicit test of the generated ics-text.

A question for this specific test: What happens if a project starts in 2021 and ends in 2022? Will/should it be exported?

Reply:
It would be expected that the start date is used to select projects from this or last year. At least this is what the filters caption says. I'll write a test for it.

Reply:
By trying to test the different start dates I stumbled over an issue with the timezones. Despite the converstion from the datetime object to the current timezone, the datetime gets handled as UTC by the ics-package. This seems to be a known Issue and open for more than 2 years.

https://icspy.readthedocs.io/en/stable/misc.html?highlight=time%20zone#datetimes-are-converted-to-utc-at-parsing-time

https://github.com/ics-py/ics-py/issues/245

https://github.com/ics-py/ics-py/issues/245#issuecomment-834179833

Reply:
There seems to be a [Pre-Release](https://pypi.org/project/ics/0.8.0.dev0/), but I am not sure whether this is a good idea to use it in this project.